node/src/database/config_dumper.rs

```diff
@@ -669,12 +669,10 @@ mod tests {
 
     fn assert_null(key: &str, map: &Map<String, Value>) {
         assert!(map.contains_key(key));
-        match map
+        let value = map
             .get(key)
-            .unwrap_or_else(|| panic!("record for {} is missing", key))
-        {
-            value => assert!(value.is_null()),
-        }
+            .unwrap_or_else(|| panic!("record for {} is missing", key));
+        assert!(value.is_null())
```

Comment:
Maybe you could provide a message to this macro which prints if it fails that would print the presented value if it was NOT null. It can make fixing failing tests easier.